grip_api/src/grip_api/abstract_graphics/abstract_socket.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
This file is going to be modified and completed in the next PR that will refactor the graphical counterparts of sockets

Reply:
So please ignore for now